articles/azure-resource-manager/management/tag-resources.md

```diff
@@ -72,6 +72,7 @@ This can result in some tools, like the Azure portal, to show the tag key twice.
 The following limitations apply to tags:
 
 * Not all resource types support tags. To determine if you can apply a tag to a resource type, see [Tag support for Azure resources](tag-support.md).
+* Each resource type may have its own unique considerations when working with tags. For instance, tags on virtual machine extensions can only be updated when the VM is running. If you encounter an error message while attempting to update a tag, please carefully follow the instructions provided in the error message.
```

Comment:
Recommended edit - Each resource type may have specific requirements when working with tags. For example, tags on virtual machine (VM) extensions can only be updated when the VM is running. If you receive an error message while trying to update a tag, follow the instructions in the message.